src/main/scala/treadle/executable/SymbolTable.scala

```diff
@@ -443,24 +453,51 @@ object SymbolTable extends LazyLogging {
               args.foreach { arg =>
                 addDependency(printSymbol, expressionToReferences(arg))
               }
-              lastPrintfInMOdule.get(module) match {
+              lastPrintfInModule.get(module) match {
                 case Some(lastPrintfSymbol) =>
                   addDependency(printSymbol, Set(lastPrintfSymbol))
                 case _ =>
               }
-              lastPrintfInMOdule(module) = printSymbol
+              lastPrintfInModule(module) = printSymbol
 
             case _ =>
               throw TreadleException(s"Can't find clock for $print")
           }
+
+        case verify @ Verification(Formal.Cover, info, clockExpression, predicateExpression, enableExpression, _) =>
+          /* do something good here */
+          getClockSymbol(clockExpression) match {
+            case Some(_) =>
+              val verifySymbolName = makeVerifyName()
+              val verifySymbol =
+                Symbol(verifySymbolName, IntSize, UnsignedInt, WireKind, 1, 1, UIntType(IntWidth(1)), info)
+              addSymbol(verifySymbol)
+
+              verifyCardinal += 1
+              verifyToVerifyInfo(verify) = VerifyInfo(verifySymbol, verifyCardinal)
+              addDependency(verifySymbol, expressionToReferences(clockExpression))
+              addDependency(verifySymbol, expressionToReferences(enableExpression))
+              addDependency(verifySymbol, expressionToReferences(predicateExpression))
+
+              lastVerifyInModule.get(module) match {
+                case Some(lastPrintfSymbol) =>
```

Comment:
I assume the `lastPrintfSymbol` is a copy and paste error.

Could you also elaborate on why you need to add a dependency on the previous cover statement?

Reply:
Yes, its a CAP typo. That code probably isn't necessary, it was derived from printf's very specifically caring about the order of evaluation. I guess that wouldn't apply here

Reply:
Thinking about it more, I can't think of why the order of coverage statement evaluation should matter. Can you. If not I'll just remove that code

Reply:
It should not matter. For asserts it would though since two of them might be violated in the same cycle.

Reply:
Fixed